README.md

```diff
@@ -37,6 +37,112 @@ The following are prerequisites to use Azure DevOps extension.
 
 Conjur (OSS or Enterprise or Cloud) and the Conjur CLI are installed in the environment and running in the background.
 
+This section describes how to set up the API Authentication for Conjur OSS or Enterprise
+
+1. Define the API Authentication policy
+
+- Create a policy that defines the API Authentication, for example BotApp.
+```yaml
+- !policy
+  id: BotApp
+  body:
+    # Define a human user, a non-human identity that represents an application, and a secret
+  - !user Dave
+  - !host myDemoApp
+  - &variables
+      - !variable secretVar
+      - !variable some-other-secret
+  - !permit
+    # Give permissions to the human user to update the secret and fetch the secret.
+    role: !user Dave
+    privileges: [read, update, execute]
+    resource: *variables
+  - !permit
+    # Give permissions to the non-human identity to fetch the secret.
+    role: !host myDemoApp
+    privileges: [read, execute]
+    resource: *variables
+```
+
+- Save the policy as BotApp.yml, and load it to root:
+
+```
+     conjur policy load -b root -f /path/to/file/authn-iam.yml
```

Comment:
Is this the correct file for this example? authn-iam.yml?

Reply:
it has to be /path/to/file/BotApp.yml